frontend/src/metabase-types/api/writeback.ts

```diff
@@ -53,30 +56,18 @@ export type WritebackAction = WritebackActionBase & (QueryAction | HttpAction);
 
 export type ParameterMappings = Record<ParameterId, ParameterTarget>;
 
-type ParameterForActionExecutionBase = {
-  type: string;
-  value: string | number;
+export type ParametersForActionExecution = {
+  [id: ParameterId]: string | number;
 };
```

Comment:
The backend didn't need all the stuff we were sending for execution, and it also didn't need us to differentiate between `parameters` (from linked filters) and `extra_parameters` (from form input), so we are able to delete a bunch of different types, mapping code, and corresponding tests.